crates/precompile/src/blob/kzg_settings.rs

```diff
@@ -0,0 +1,34 @@
+use alloc::boxed::Box;
+use c_kzg::KzgSettings;
+use once_cell::race::OnceBox;
+
+#[rustfmt::skip]
+mod generated;
+
+/// Note: the type of this is an implementation detail, so we don't expose it in the public API.
+static GLOBAL_KZG_SETTINGS: OnceBox<KzgSettings> = OnceBox::new();
```

Comment:
I'd prefer if this would be `Arc<KzgSettings>` otherwise it would be much harder to install custom settings because KzgSettings is not Clone

Reply:
Doesn't the current OnceBox allow that by overriding it once?